cinnamon-shell/src/sass/_tweaks.scss

```
@@ -0,0 +1,145 @@
+// Yaru: any modifications to Mint's original cinnamon theme
+// may be added in this file. We will use this file override
+// any settings that are contradictory to project goals of yaru
+// except for the colours. Any modifications in colors should be added
+// to _colors.scss
+
+// hover: we want the hover bg to be a visible gray for dark and light shell themes
+$hover_bg_color: transparentize($fg_color, 0.9);
+$hover_fg_color: $fg_color;
+$hover_borders_color: lighten($borders_color, if($variant=='light', 5%, 3%));
+
+// active: we want the active bg to be a visible gray for dark and light shell themes
+$active_bg_color: transparentize($fg_color, 0.8);
+$active_fg_color: darken($fg_color, if($variant=='light', 5%, 3%));
+$active_borders_color: darken($borders_color, if($variant=='light', 5%, 3%));
+
+//
+// Calendar
+//
+.calendar-day-base {
+  border-radius: 999px;
+}
+
+//
+// Check Boxes: checkboxes and radios are 24x24px
+//
+.check-box,
+.radiobutton {
+  StBin {
+    width: 24px;
+    height: 24px;
```

Can you revert the control's size to 20px like before?

```suggestion
    width: 20px;
    height: 20px;
```